src/prefect/artifacts.py

```diff
@@ -195,6 +195,21 @@ async def format(self) -> float:
         return self.progress
 
 
+class ImageArtifact(Artifact):
+    """
+    An artifact that will display an image from a publicly accessible URL in the UI.
+
+    Arguments:
+        image_url: The URL of the image to display.
+    """
+
+    image_url: str
+    type: Optional[str] = "image"
+
+    async def format(self) -> str:
```

Comment:
this method seems a bit weird to me. i see theres an `async def format` on the parent class, maybe we could add a docstring here to say why we have this / why its async?